src/plots/get_data.js

```diff
@@ -69,6 +69,7 @@ exports.getModuleCalcData = function(calcdata, arg1) {
     for(var i = 0; i < calcdata.length; i++) {
         var cd = calcdata[i];
         var trace = cd[0].trace;
+        // N.B. 'legendonly' traces do not make it pass here
```

Comment:
~pass~ past

Reply:
fixed in https://github.com/plotly/plotly.js/pull/2860/commits/89aebd1737bb3fe9b82ff61b8793050901eeeba7